### PR TITLE
adminguide: updates for v0.38.0

### PR DESCRIPTION
**Pull request by grondo**
This is a WIP which adds documentation of a couple new features in the flux-core v0.38.0 release.

Just parking this here for now in case someone wants to correct, amend, emend, or expand on these two new sections thus far.

I guess we should also update all the "notes" to reference the new release as well.

### CODE REVIEW

**Comment by garlick** [comment]
I pushed a few more updates - hope that's ok.

**Comment by grondo** [comment]
Yes, that was what I was hoping! I'll take a look in the morning

File: adminguide.rst [hunk 1]
```diff
@@ -498,6 +498,8 @@ TOML table, as shown in the example below:
 
 .. code-block:: toml
 
+  # /etc/flux/system/conf.d/ingest.toml
```
Comment (grondo) [inline adminguide.rst:501]: This seems fine, but I was wondering if our implication that the toml config for every subsystem has to be in a different file is going to drive sysadmins crazy. Is there some benefit to implying that?

Comment (garlick) [reply adminguide.rst:501]: Yeah good point.   I was just going for consistency here, but I'm open to changing the overall presentation.

Comment (grondo) [reply adminguide.rst:501]: Yeah, I was more thinking out loud at this point

**Comment by grondo** [comment]
Agreed. Setting MWP.

**Comment by garlick** [comment]
@mergify-io refresh

**Pull request merged**